### PR TITLE
Remove circleci and update readme

### PR DESCRIPTION
**Pull request by chiaberry**
As part of accomplishing https://github.com/cityofaustin/atd-data-tech/issues/8788, I have deprecated the circleCI integration. 

Due to the lack of active development on this repo I've chosen to not create a github action to push changes to s3. Changes should be manually uploaded.

### CODE REVIEW

**Review by frankhereford — APPROVED** [review]
Thanks for the removal of the CircleCI integration. If this repo comes back into frequent updates, we can look at making a github action for CI to upload updated versions of the JS to S3. But for the time being, this is just what we need. 

Thanks for bringing this over into our AWS domain @chiaberry and the discovery to learn how this worked.

File: knack/README.md [hunk 1]
```diff
@@ -3,3 +3,11 @@
 This directory is for version control of the jQuery code related to the Knack lat/lon picker UI that gets edited and deployed in our testing instance of Data Tracker.
 
 https://builder.knack.com/atd/21-may-2019-test-signs-migration-atd-data-tracker#settings/api
+
+
+_Note_
```
Comment (mddilley) [inline knack/README.md:8]: Yay docs!

**Review by mddilley — APPROVED** [review]
Awesome! Now, I understand what CircleCI was doing, and I agree that removing the integration is a safe choice since this code hasn't been touched in longgggg time. Thanks for migrating this and keeping the app in working order! 🙏

**Review by johnclary — APPROVED** [review]
thanks @chiaberry!

⛵

**Pull request merged**